src/views/Products/ProductsDetailsById.vue

```diff
@@ -46,11 +46,25 @@
           />
         </div>
 
-        <strong>Part of Sale: </strong>
-        <part-of-product
-          :partOfProduct="currentProductInfo.partOfSale"
-          routerPreFix="sales"
-        />
+        <div>
+          <strong>Part of Sale: </strong>
+          <part-of-product
```

Comment:
This name looks wrong but I guess this is what it was named initially. Shouldn't it be part-of-sale or I am missing something

Reply:
yes, in the way we use it is wrong, but its a component we are using twice, once for contentOf and once for partOfSale